xcm/xcm-builder/src/fungibles_adapter.rs

```diff
@@ -178,7 +178,7 @@ impl<
 			// This is an asset whose teleports we track.
 			let checking_account = CheckingAccount::get();
 			Assets::can_withdraw(asset_id, &checking_account, amount)
-				.into_result()
+				.into_result(false)
```

Comment:
Setting `keep_alive: false` produces the same result as before but is it correct?